spec/acceptance/linux_spec.rb

```diff
@@ -1,53 +1,37 @@
 # run a test task
 require 'spec_helper_acceptance'
 
-describe 'linux service task', unless: fact('osfamily') == 'windows' do
+describe 'linux service task', unless: os[:family] == 'windows' do
   include Beaker::TaskHelper::Inventory
   include BoltSpec::Run
 
-  def module_path
-    RSpec.configuration.module_path
-  end
-
-  def config
-    { 'modulepath' => module_path }
-  end
-
-  def inventory
-    hosts_to_inventory
-  end
-
-  def run(params)
-    run_task('service::linux', 'default', params, config: config, inventory: inventory)
-  end
-
   package_to_use = 'rsyslog'
   before(:all) do
-    if fact('osfamily') == 'RedHat' && fact('operatingsystemrelease') < '6'
-      run('action' => 'stop', 'name' => 'syslog')
+    if os[:family] == 'redhat' && os[:release].to_i < 6
+      task_run('service::linux', 'action' => 'stop', 'name' => 'syslog')
     end
     apply_manifest_on(default, "package { \"#{package_to_use}\": ensure => present, }")
```

Comment:
As previous, see https://github.com/puppetlabs/bolt/blob/master/lib/bolt_spec/run.rb#L50 to use BoltSpec apply_manifest.